.gitignore

```diff
@@ -49,7 +49,7 @@ node_modules/
 
 # IntelliJ IDEA
 .idea
-.ijwb # Bazel
+.ijwb
```

Comment:
I wonder if this is down to Git version? This works fine for me. (Ship it though.)

Reply:
Could be.

Reply:
Official docs only mention lines starting with # as comments.